config/psyclone.cfg

```diff
@@ -71,12 +71,19 @@ access_mapping = gh_read: read, gh_write: write, gh_readwrite: readwrite,
 # remove the need for a halo exchange call.
 COMPUTE_ANNEXED_DOFS = false
 
-# Specify default kind (precision) for LFRic data types
+# Specify supported Fortran datatypes
+supported_fortran_datatypes = real, integer, logical
+
+# Specify default kind (precision) for the supported LFRic data types
 default_kind = real: r_def, integer: i_def, logical: l_def
 
 # Specify whether we generate code to perform runtime correctness checks
 RUN_TIME_CHECKS = false
 
+# Number of ANY_SPACE and ANY_DISCONTINUOUS_SPACE function spaces
+NUM_ANY_SPACE = 10
+NUM_ANY_DISCONTINUOUS_SPACE = 10
```

Comment:
I would like to keep the default as it is in PSyclone since all our testing and the local copy of the LFRic infrastructure is adjusted to it. I think we don't really need to follow LFRic in the number of spaces as it is a temporary thing for the massive Physics kernels that is supposed to come out once we have the proper solution for function spaces and we don't need to test or work with in PSyclone.

Reply:
Yes, that makes sense. Thanks for explaining.